ci/testnet-automation.sh

```diff
@@ -52,14 +52,14 @@ launchTestnet() {
   declare q_mean_tps='
     SELECT round(mean("sum_count")) AS "mean_tps" FROM (
       SELECT sum("count") AS "sum_count"
-        FROM "testnet-automation"."autogen"."counter-bank-process_transactions-txs"
```

Comment:
can we compute the average bank tps from all the validators?

Reply:
We can add a new counter on the dashboard for that.